spec/support/shared_context_for_module_methods_via_class_scope.rb

```diff
@@ -13,4 +15,6 @@ class << self
       end
     end
   end
+
+  let(:module_with_memo) { @_module_with_memo } # rubocop:disable RSpec/InstanceVariable
```

Comment:
Does `module_with_memo` need the same Module-level resetting of state that the `class_with_memo` needed?

Reply:
I haven't looked into why, but it does not.